angular-part/src/app/pages/news-page/components/news-card/news-card.component.ts

```diff
@@ -22,4 +23,8 @@ export class NewsCardComponent implements OnInit {
     this.edit.emit(this.item);
   }
 
+  public deleteItem() {
+    this.delete.emit(this.item);
```

Comment:
What do you think about implementing confirmation popup before sending event to parent component?